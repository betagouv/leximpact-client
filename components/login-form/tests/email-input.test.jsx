/* eslint
    indent: [2, 2],
    semi: [2, "always"],
    react/jsx-indent: [2, 2],
    react/jsx-indent-props: [2, 2]
    max-nested-callbacks: [2, { "max": 4 }]
*/
import { createShallow } from "@material-ui/core/test-utils";

import EmailInput from "../email-input";

describe("components | login-form | email-input", () => {
  let shallow;
  beforeAll(() => {
    shallow = createShallow();
  });
  describe("snapshot", () => {
    it("doit correspondre avec les props requises", () => {
      const props = {
        domains: ["@any-domain"],
      };
      const wrapper = shallow(<EmailInput {...props} />);
      expect(wrapper).toBeDefined();
      expect(wrapper).toMatchSnapshot();
    });
  });
});
