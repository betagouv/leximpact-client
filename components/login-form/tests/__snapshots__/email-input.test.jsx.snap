// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components | login-form | email-input snapshot doit correspondre avec les props requises 1`] = `
<EmailTextInput
  classes={
    Object {
      "fieldsContainer": "EmailTextInput-fieldsContainer-3",
      "formGroup": "EmailTextInput-formGroup-1",
      "formLabel": "EmailTextInput-formLabel-2",
      "inputContainer": "EmailTextInput-inputContainer-4",
      "inputError": "EmailTextInput-inputError-7",
      "inputField": "EmailTextInput-inputField-6",
      "inputRoot": "EmailTextInput-inputRoot-5",
      "selectRoot": "EmailTextInput-selectRoot-8",
    }
  }
  domains={
    Array [
      "@any-domain",
    ]
  }
/>
`;
